guides/office/toilet.md

```diff
@@ -0,0 +1,29 @@
+# Toilet Etiquette
+
+Going to the toilet should not be something to be dreaded. New research from [The European Fecal Standards & Measurements Board](http://www.efsmi.org/) has shown that going to the toilet and being greeted by feces which register over one couric can cause long term damage to mental health.
+
+No one is too good to put in some hard manual labour and clean the toilets when necessary. Let's keep the toilets sparkling clean, together.
+
+## Toilet tenet
+
+When using the toilets, please make sure you perform the following:
+
+ - Flush the toilet after use, half flush if you can to save water (and the planet)
+ - If any fecal matter remains inside the toilet bowl, please use the toilet brush to clean it
+ - If the fecal matter is outside the toilet bowl, please seek medical attention
```

Comment:
This probably can't stay in here as it is.

Reply:
Agree, need to rewrite.  do we need toilet guides?

Reply:
Can we remove the toilet guide please? :) I'll call it out on slack (if I have to). Unless it becomes a problem I would rather not have this guide.